src/test/java/cucumber/steps/OneDriveStepDefinitions.java

```diff
@@ -0,0 +1,36 @@
+package cucumber.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import pages.overDrivePages.OverDriveBooksCollectionsPage;
+import pages.overDrivePages.OverDriveHistoricalBooksPage;
+import pages.overDrivePages.OverDriveHomePage;
+import service.DriverSingleton;
+
+public class OneDriveStepDefinitions {
+    @Given("User navigates OverDrive library home page")
+    public void openOverDriveHomePage() {
+        new OverDriveHomePage(DriverSingleton.getDriverInstance()).pageOpener();
+    }
+
+    @When("^User chooses Browse Collection item$")
```

Comment:
You can remove signs like ^ and $. It is old style of bdd steps description.

Reply:
done